README.md

```diff
@@ -84,7 +84,10 @@ persist data. Upon initialize, the SDK registers listeners for your application'
 to gracefully manage background processes.
 
 `Klaviyo.initialize()` **must** be called before any other SDK methods can be invoked. We recommend initializing from 
-the earliest point in your application code, such as the `Application.onCreate()` method.
+the earliest point in your application code, such as the `Application.onCreate()` method If you are
```

Comment:
Wondering if we should make this callout less prominent, since it will hopefully be quite an edge case. Maybe as a note below

Reply:
Couldn't write this in a comment suggestion, put it in PR instead :
https://github.com/klaviyo/klaviyo-android-sdk/pull/242